core/tests/widgets/test_imageview.py

```diff
@@ -171,3 +183,19 @@ def test_rehint_empty_image(params):
     assert width == 0
     assert height == 0
     assert aspect_ratio is None
+
+
+def test_as_image_toga():
+    toga_img = toga.Image(ABSOLUTE_FILE_PATH)
+    imageview = toga.ImageView(toga_img)
+    toga_img_2 = imageview.as_image()
+    assert toga_img_2.size == (60, 40)
+
+
+def test_as_image_pil():
+    imageview = toga.ImageView(ABSOLUTE_FILE_PATH)
+
+    # Doesn't work with dummy backend
+    with pytest.raises(PIL.UnidentifiedImageError):
```

Comment:
As before - we should probably make this a live test to make sure the PIL interface actually works.